# Pose message
uint8 REST = 1
uint8 FIST = 2
uint8 WAVE_IN = 3
uint8 WAVE_OUT = 4
uint8 FINGERS_SPREAD = 5
uint8 THUMB_TO_PINKY = 6
uint8 UNKNOWN = 0
uint8 pose
